llmfoundry/utils/config_utils.py

```diff
@@ -592,7 +592,24 @@ def _process_data_source(
     elif 'hf_name' in dataset:
         hf_path = dataset['hf_name']
         backend, _, _ = parse_uri(hf_path)
-        if backend:
+        unsupported_file = True
+        if hf_path.startswith('dbfs:'):
+            assert cfg_split
+            from llmfoundry.data.finetuning.tasks import SUPPORTED_EXTENSIONS
+            possible_files = [
+                f'{cfg_split}{ext}' for ext in SUPPORTED_EXTENSIONS
+            ]
+            for file in possible_files:
+                path = os.path.join(hf_path[len('dbfs:'):], file)
+                if _verify_uc_path(path):
+                    data_paths.append(('uc_volume', path, true_split))
+                    unsupported_file = True
```

Comment:
this should be set to false here, right?

Reply:
line 606 should be False, if we detect that the file is supported, we should cast a warning